argocd/argocd-app.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
Is this file really needed? When seeing in context with the argo-file in the infrastructure repo, it seems like the argo-file in the infrastructure repo could point directly to the infrastructure-folder in this repo?

Reply:
It could, but then all resources would belong to a single app. Instead of having this:
<img width="1058" alt="image" src="https://github.com/openearthplatforminitiative/geocoder-api/assets/22095727/261d9868-aa55-408e-a075-4177f4714a69">
We would only see the "apps"-app. We would then lose the overview of which specific app needs syncing and the health status of specific apps. It would also make it a bit more difficult to sync only one specific app

Reply:
I am however ok with moving this file to the infrastructure-repo. My first thought was that I should move application-specific code as close to the application-repo as possible. However, this file probably won't need much customization.

Reply:
Let's keep it the way it is now.